src/components/EventDetails/EventDetails.scss

```diff
@@ -0,0 +1,31 @@
+.description {
+  margin-bottom: 1em;
+  font-size: 1.3rem;
+  color: var(--active-tab-color);
+}
+
+.info-details {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center; 
+  gap: 1em;
+}
+
+.info-details .list-leader {
+  flex-basis: 230px;
+  flex-shrink: 0;
+}
+
+@media (max-width: 680px) {
+  .info-details {
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0;
+    margin-bottom: 1em;
+    overflow-wrap: anywhere;
```

Comment:
I'm never using anything other than  'anywhere' in an MVP again

Reply:
🤣